<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthRegForm</class>
 <widget class="QWidget" name="AuthRegForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>233</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>10</y>
     <width>301</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="0">
       <widget class="QLabel" name="labelPassword">
        <property name="text">
         <string>Password:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLineEdit" name="lineEditEmail"/>
      </item>
      <item row="6" column="0">
       <widget class="QLineEdit" name="lineEditPassword"/>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="lineEditLogin"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="labelEmail">
        <property name="text">
         <string>Email:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelLogin">
        <property name="text">
         <string>Login:</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButtonAuth">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(187, 229, 255);</string>
          </property>
          <property name="text">
           <string>Авторизация</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonRegistration">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(187, 229, 255);</string>
          </property>
          <property name="text">
           <string>Регистрация</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonChange">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(187, 229, 255);</string>
          </property>
          <property name="text">
           <string>У меня нет аккаунта</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
